bin/arnold

```diff
@@ -0,0 +1,832 @@
+        log error ".team file does not exist"
+        exit 12
+    fi
+
+    SECRETS_PATH=$(_get_secrets_directory)
+
+    if [[ ! -d ${SECRETS_PATH} && ${skip} -eq 1 ]]; then
+        log error "The 'secrets' directory does not exist and skip option is being used"
+        log error "Vault password file will not be generated"
+        return
+    fi
+
+    # Make a recipients list from emails in $TEAM_FILE. Lines starting with a
+    # '#' will be treated as comments.
+    recipients=$(grep -vE "^\\s*#" "${TEAM_FILE}" | sed 's/\s*\(\S*@\S*\)\s*/-r \1 /g' | tr -d '\n')
+
+    GPG_PASSWORD_FILE=$(_get_vault_password_file)
+
+    if [[ -z "${recipients}" ]]; then
+        log error "No recipient provided, please populate expected .team file"
+        exit 10
+    fi
+
+    if [[ -e "${GPG_PASSWORD_FILE}" && $force -eq 0 ]]; then
+        log error "Vault password file already exists: ${GPG_PASSWORD_FILE}"
+        exit 11
+    else
+        # Generate a 128 characters long random password, encrypt it and save it
+        # to $GPG_PASSWORD_FILE
+        #
+        # shellcheck disable=SC2086
+        # (☝ we want word splitting on the recipients variable)
+        gpg -a --gen-random 2 128 | gpg --yes -e ${recipients} -o "${GPG_PASSWORD_FILE}"
+    fi
+
+    log success "Generated ansible vault password file: ${GPG_PASSWORD_FILE}"
+}
+
+
+# _set_tray_vars: set in-development tray variables
+function _set_tray_vars() {
+
+  if [[ ! -f ${ARNOLD_TRAY_FILE} ]]; then
+    log info "Tray file is missing, supposing you are not developing a local tray"
+    return
+  fi
+
+  ARNOLD_TRAY_PATH=$(${YQ} r "${ARNOLD_TRAY_FILE}" source.path)
+  export ARNOLD_TRAY_PATH
+  log debug "Tray path: ${ARNOLD_TRAY_PATH}"
+
+  ARNOLD_TRAY_NAME=$(${YQ} r "${ARNOLD_TRAY_FILE}" metadata.name)
+  export ARNOLD_TRAY_NAME
+  log debug "Tray name: ${ARNOLD_TRAY_NAME}"
+}
+
+
+# _docker_run: wrap docker run command
+#
+# usage: _docker_run [OPTIONS] [COMMAND]
+#
+#   COMMAND: command to be run in Arnold container (_e.g._ deploy)
+#
+# NB: the docker run call overrides Arnold's default `${CUSTOMER_DIR}` directory
+# by Matsuo's definitions.
+function _docker_run() {
+
+    declare extras_volumes="\
+      -v ${COMMON_DIR}:/app/${RELATIVE_COMMON_DIR} \
+      -v ${CUSTOMER_DIR}:/app/${RELATIVE_CUSTOMER_DIR}"
+
+    # Get password from gpg-encrypted vault password
+    ANSIBLE_VAULT_PASSWORD=$(_get_vault_password)
+    export ANSIBLE_VAULT_PASSWORD
+
+    if [[ DEV_MODE -eq 1 ]]; then
+      _set_tray_vars
+
+      if [[ -e ${ARNOLD_TRAY_PATH} ]]; then
+        extras_volumes="${extras_volumes} \
+          -v ${PWD}/${ARNOLD_TRAY_PATH}:/app/apps/${ARNOLD_TRAY_NAME}"
+        log info "Development mode activated for tray: ${ARNOLD_TRAY_NAME}"
+      else
+        extras_volumes="-v ${PWD}:/app"
+        log info "Full development mode activated"
+      fi
+    fi
+
+    # shellcheck disable=SC2086
+    docker run --rm -ti \
+        -u "$(id -u)" \
```

Comment:
What do you think about adding a variable like this : 
```
DOCKER_USER="${DOCKER_USER:-$(id -u)}"
```

and use it as a value for the `-u` argument  ?

Reply:
In fact, this is breaking the CI as the DOCKER_USER already exists (as a secret)... :sweat_smile: